app/controllers/finders_controller.rb

```diff
@@ -11,6 +11,16 @@ class FindersController < ApplicationController
   ATOM_FEED_MAX_AGE = 300
 
   def show
+    ab_test = GovukAbTesting::AbTest.new(
+      "SearchClusterABTest",
+      dimension: 24601, # todo: talk to a PA
+      allowed_variants: %w(A B),
```

Comment:
We've got some evidence that our method of assigning people to variants is not fool proof (see [A/B/C method](https://docs.google.com/document/d/1ysC8NYkcXIHUDAK0E7Nz-KQ7-TYlIcABt8-ca99QTHo/edit#heading=h.iwqog86sbciv).  We got around the discrepancy by assigning A as the default, B as the control (behaving the same as A) and C as the variant.  I need to work out better ways of explaining this

Reply:
Suganya was the discoverer of this